README.md

```diff
@@ -212,11 +213,24 @@ Also, there is a shortened version of the raw data - the "SP500" csv file. Since
 
 In addition, as part of the data preparation, we have create the sequences of the data. They were created by building a custom sequence distributor (see sequence_distributor.py) to slice data into the small windows of 30 days each, which will be used as predictors, and the slices with the span of 5 days - the target values that need to be predicted. The predictor and target data were scaled separetelly using different approaches. So overall we have created 14,832 sequences. These sequences were used as an inputs for our Time Series Neural Networks: TDNN, LSTM, RNN.
 
+**Pipeline** 
+
+Speaking of analyzing the raw visual data,  we had to create a certain pipeline to put it into the proper format and then make forecasts based on that. First of all, simply inputting the whole sequence of candle plots and asking the models to make a forecast -(continue the picture) is not the best approach for our specific task. We need to detach the individual candles in the first place, remove the background noise (for instance the grid) and so on. The illustration below demonstrates how we do it. 
+
+![Pipeline](images/pipeline_ps.png)
 
 
+
+1. We take the  images with candle plots and then pass them trough our **selector**. The purpose of the **selector** is to find and detach each individual candle into separate images. 
+2. As an output we get separated candles. 
+3. Afterwards, we pass each individual candle trough **interpreter**. The purpose of interpreter is to convert  individual candles into more convenient numeric representation which we have already mentioned. 
```

Comment:
interpretatOr
that should be with a typo

Reply:
Typora shows a mistake when I type intepretator. maybe I should update the pic then?

Reply:
that would be gret!